yahpo_gym/setup.py

```diff
@@ -27,6 +27,7 @@
     description="Inference module for YAHPO Gym",
     long_description=long_description,
     license="Apache-2.0",
+    python_requires="<=3.11.9",
```

Comment:
why is that necessary?

Reply:
This is a result of https://github.com/slds-lmu/yahpo_gym/issues/91

Reply:
not accept